forms/common-themes/src/main/resources/theme/keycloak/login/resources/css/login.css

```diff
@@ -122,8 +122,8 @@
 }
 
 #kc-login {
```

Comment:
This is not correct. As the login button is on the left side there's no need for the float. Also, by adding the margin you are adding a bigger margin that specified in PatternFly.

We should just remove '#kc-login {....}' completely.